src/families/multicall/transfer.js

```diff
@@ -42,7 +42,7 @@ export default class Transfer extends BaseTask {
                     amount: new ArgsBig("0", "0", null, "yocto")                    
                 }    
             ),
-            gas: new ArgsNumber(actions?.gas ?? toGas(3), 1, toGas(300), "gas"),
+            gas: new ArgsBig(actions?.gas ?? toGas("3"), 1, toGas("300"), "gas"),
             depo: new ArgsBig("0", "0", "0", "yocto")
```

Comment:
Please make this NEAR and update the order of unit options below too.

- [ ] Default value in `"NEAR"`
- [ ] Options for units `["NEAR","yocto"]` and `["Tgas","gas"]`

Maybe that's not part of your ticket, but if you could make the entire code base stick to these rules, that would be great.
To avoid too many comments, I'll just put this here once, please also correct this in the other cards.

Reply:
Fixed